support/doc/tools.md

```diff
@@ -45,22 +47,24 @@ $ npm run build:server
 You can use this script to import videos from all [supported sites of youtube-dl](https://rg3.github.io/youtube-dl/supportedsites.html) into PeerTube.  
 Be sure you own the videos or have the author's authorization to do so.
 
-
-```
-$ cd ${CLONE}
-$ node dist/server/tools/import-videos.js -u "PEERTUBE_URL" -U "PEERTUBE_USER" --password "PEERTUBE_PASSWORD" -t "TARGET_URL"
+```sh
+$ node dist/server/tools/import-videos.js \
+    -u "PEERTUBE_URL" \
+    -U "PEERTUBE_USER" \
+    --password "PEERTUBE_PASSWORD" \
+    -t "TARGET_URL"
 ```
 
- * PEERTUBE_URL : the full URL of your PeerTube server where you want to import, eg: https://peertube.cpy.re
- * PEERTUBE_USER : your PeerTube account where videos will be uploaded
- * PEERTUBE_PASSWORD : password of your PeerTube account (if ommited, you will be prompted for)
```

Comment:
Why did you remove `(if ommited, you will be prompted for)`?

Reply:
A mistake :/